### PR TITLE
chore: Update documentation for custom smart contracts

### PR DESCRIPTION
**Pull request by dwertent**
Changing from a list to paragraphs because each point contains a lot of data, which makes it more readable and easier to reference when needed

### CODE REVIEW

File: doc-site/docs/tutorials/custom_contracts/index.md [hunk 1]
```diff
@@ -12,25 +12,34 @@ FireFly's unified API creates a consistent application experience regardless of
...
 
-- **Event Subscription**: While an event listener tells FireFly to keep track of certain events emitted by the blockchain, an event subscription tells FireFly to relay those events to the client application. Each subscriptions represents a stream of events that can be delivered to a listening client with various modes of delivery with at-least-once delivery guarantee.
+### Event Subscription
+
+An event listener in FireFly tracks specific blockchain events, while an event subscription directs FireFly to send those events to the client application. Each subscription creates a stream of events that can be delivered to the client with various delivery options, ensuring an at-least-once delivery guarantee.
 
-This is exactly the same as listening for any other events from FireFly. For more details on how Subscriptions work in FireFly you can read the [Getting Started guide to Listen for events](../events.md).
+This process is similar to listening for any other events in FireFly. For further details on how subscriptions work, refer to the [Getting Started guide to listening for events](../events.md).
```
Comment (EnriqueL8) [inline doc-site/docs/tutorials/custom_contracts/index.md:42]: I prefer the previous wording of this phrase and keep `Subscriptions` capital is highlighting that it's a resource in the system

File: doc-site/docs/tutorials/custom_contracts/index.md [hunk 2]
```diff
@@ -12,25 +12,34 @@ FireFly's unified API creates a consistent application experience regardless of
...
+### Contract Interface
+
+FireFly defines a common, blockchain agnostic way to describe smart contracts. This is referred to as a Contract Interface. A contract interface is written in the FireFly Interface (FFI) format. It is a simple JSON document that has a name, a namespace, a version, a list of methods, and a list of events.
 
 For more details, you can also have a look at the [Reference page for the FireFly Interface Format](../../reference/firefly_interface_format.md).
 
-For blockchains that offer a DSL describing the smart contract interface, such as Ethereum's ABI (Application Binary Interface), FireFly offers a convenience tool to convert the DSL into the FFI format.
+For blockchains that offer a DSL describing the smart contract interface, such as Ethereum's ABI (Application Binary Interface), FireFly offers a convenience tool to convert the DSL into the FFI format. <!-- TODO: Add link to the convenience tool docs -->
```
[the remark this answers is not in the document]
Comment (dwertent) [reply doc-site/docs/tutorials/custom_contracts/index.md:21]: Where can I find the convenience tool docs?

[the remark this answers is not in the document]
Comment (EnriqueL8) [reply doc-site/docs/tutorials/custom_contracts/index.md:21]: I'm not sure we have this convenience tool

[the remark this answers is not in the document]
Comment (EnriqueL8) [reply doc-site/docs/tutorials/custom_contracts/index.md:21]: It's just the `/contracts/interfaces/generate` API

File: doc-site/docs/tutorials/custom_contracts/index.md [hunk 3]
```diff
@@ -12,23 +12,32 @@ FireFly's unified API creates a consistent application experience regardless of
...
+### Contract Interface
+
+FireFly defines a common, blockchain agnostic way to describe smart contracts. This is referred to as a Contract Interface. A contract interface is written in the FireFly Interface (FFI) format. It is a simple JSON document that has a name, a namespace, a version, a list of methods, and a list of events.
 
 For more details, you can also have a look at the [Reference page for the FireFly Interface Format](../../reference/firefly_interface_format.md).
 
-For blockchains that offer a DSL describing the smart contract interface, such as Ethereum's ABI (Application Binary Interface), FireFly offers a convenience tool to convert the DSL into the FFI format.
+For blockchains that offer a DSL describing the smart contract interface, such as Ethereum's ABI (Application Binary Interface), FireFly offers an API to [convert the DSL into the FFI format](../custom_contracts/ethereum.md#the-firefly-interface-format).
```
Comment (dwertent) [inline doc-site/docs/tutorials/custom_contracts/index.md:21]: @EnriqueL8 I think this is more accurate: 
`FireFly offers a convenience tool to convert the DSL into the FFI format` -> `FireFly offers an API to convert the DSL into the FFI format`

Comment (EnriqueL8) [reply doc-site/docs/tutorials/custom_contracts/index.md:21]: yeah awesome

**Review by EnriqueL8 — APPROVED** [review]
LGTM

**Pull request merged**